pkg/mcufont/patches/0002-decoder-Make-unused-parameters-explicit.patch

```diff
@@ -0,0 +1,37 @@
+From 48c80d4463ab7cb000e5c84a6796f77a207174f3 Mon Sep 17 00:00:00 2001
+From: Koen Zandberg <koen@bergzand.net>
+Date: Sun, 11 Jun 2023 22:07:09 +0200
+Subject: [PATCH 2/2] decoder: Make unused parameters explicit
```

Comment:
since those have already been merged, you can drop the patches :smiley:

Reply:
Yup, I'm aware :smile:

Reply:
Will do that after crafting the comparison patch for msp430